fips202/fips202.c

```diff
@@ -1,5 +1,7 @@
-// Copyright (c) 2024 The mlkem-native project authors
-// SPDX-License-Identifier: CC0-1.0
+/*
+ * Copyright (c) 2024 The mlkem-native project authors
```

Comment:
I assume you did not do this by hand -- how did you automate the reformatting?

Reply:
I replaced all copyright headers in a one-off replace. 
Then I replaced all one line comments `//` with `/* */` using a reg-ex. 
Then. I went through all other comments manually and replaced them manually depending on what makes sense. 

Finally (after your review), I went through all multi-line comments manually and replaced
```
/* XXX */ 
/* YYY */ 
```

with 
``` 
/* 
  * XXX
  * YYY
  */
  ```